ExampleMod/Common/Systems/TownNPCRespawnSystem.cs

```diff
@@ -33,13 +33,11 @@ public override void LoadWorldData(TagCompound tag) {
 		}
 
 		public override void NetSend(BinaryWriter writer) {
-			writer.Write(
-				unlockedExamplePersonSpawn
-			);
+			writer.WriteFlags(unlockedExamplePersonSpawn);
 		}
 
 		public override void NetReceive(BinaryReader reader) {
-			(unlockedExamplePersonSpawn, savedExamplePerson) = reader.ReadBitsByte();
```

Comment:
Can you check ExampleMod for any other usages of `ReadBoolean`? We may as well teach `ReadFlags` as a complete replacement, because it has the same performance as `ReadBoolean`, but extends bettter to multiple bools